python/test/unit/language/test_core_amd.py

```diff
@@ -1128,13 +1128,10 @@ def kernel(X, Z, BLOCK_M: tl.constexpr, BLOCK_N: tl.constexpr, AXIS: tl.constexp
                          [(dtype, shape, perm)
                           # TODO: bfloat16
                           for dtype in ['float16', 'float32']
-                             for shape in [(64, 64), (128, 128)]
+                             for shape in [(64, 64)]
                              for perm in [(1, 0)]])
 def test_permute(dtype_str, shape, perm, device='cuda'):
     check_type_supported(dtype_str)  # bfloat16 on cc < 80 will not be tested
-    if torch.version.hip is not None:
-        if (dtype_str, shape) == ('float32', (128, 128)):
-            pytest.skip("Not supported: memory out of resource.")
```

Comment:
The `['float32', (128, 128)] ` configuration is already disabled here for ROCm stack. But your change also removes configuration `['float16', (128, 128)] `.
Is it intentional? The descritption says about only one config...